### PR TITLE
Example device

### PR DESCRIPTION
**Pull request by jboulder**

### CODE REVIEW

**Comment by jboulder** [comment]
ack'ed by robert

**Comment by jordens** [comment]
no, no. not this one. title does not match content.

File: setup.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
             "lda_controller=artiq.frontend.lda_controller:main",
             "pdq2_client=artiq.frontend.pdq2_client:main",
             "pdq2_controller=artiq.frontend.pdq2_controller:main",
+            "example_artiq_device_client=artiq.frontend.example_artiq_device_client:main",
```
Comment (sbourdeauducq) [inline setup.py]: You probably do not want to install this.

**Comment by sbourdeauducq** [comment]
Are there still things in this pull request that you would like to see merged, Joe?

**Pull request closed without merging**